.github/workflows/build-mac.yml

```diff
@@ -0,0 +1,114 @@
+name: Build installable mac distributions
+#on: workflow_dispatch
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+#    environment: release
+    strategy:
+      matrix:
+        os: [ macos-11, macos-12 ]
+
+#    permissions:
+#      contents: write
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Install Python 3
+        uses: actions/setup-python@v4.1.0
+        with:
+          cache: 'pip' # caching pip dependencies
+          python-version: 3.9
+      - run: pip install -r requirements-dev.txt
+      - run: pip install -r requirements-mac-build.txt
+      - uses: actions/checkout@v3.0.2
+        with:
+          repository: Arelle/EdgarRenderer
```

Comment:
side note: not something to figure out here, but we should update all of the build workflows to use inputs to support building plugins with more branches than just their master.

Reply:
That makes sense to me. I haven't looked into it, but it might be nice to just provide a script in arelle proper to setup all these for development and then just call that from all of the builds